src/test/java/TSibatyykEvhenii/Tests/DrugsPrice.java

```diff
@@ -0,0 +1,30 @@
+package TSibatyykEvhenii.Tests;
+
+import BasePageAndTestInit.TestInit;
+import TSibatyykEvhenii.Asserts.Alteya;
+import TSibatyykEvhenii.Pages.Alteya.AlteyaMAinPage;
+import TSibatyykEvhenii.Pages.Alteya.AlteyaPharmacy;
+import TSibatyykEvhenii.Pages.Alteya.DentistPage;
+import org.testng.annotations.Test;
+
+public class DrugsPrice extends TestInit {
+
+    @Test
+    public void checkPriceInAlteya(){
+        AlteyaMAinPage alteyaMAinPage = new AlteyaMAinPage(driver);
```

Comment:
Would You be so kind and add an empty string after the 13 string

Reply:
With pleasure

Reply:
Thanks